flow/endpoint.yaml

```diff
@@ -46,12 +46,28 @@ components:
         rx_name:
           description: |-
             The unique name of a port that is the intended receive port.
+          x-status:
+            status: deprecated
+            information: |-
+              This property is deprecated in favor of property rx_names
           type: string
           x-constraint:
           - '/components/schemas/Port/properties/name'
           - '/components/schemas/Lag/properties/name'
 
           x-field-uid: 2
+        rx_names:
+          description: |-
+            Unique names of ports that are intended receive ports.
+          type: array
+          items:
+            description: |-
+              The unique name of a port that is the intended receive port.
```

Comment:
Should we correct 'port' to 'port or lag'?

Reply:
Not seeing the updated change?